framework/src/org/checkerframework/framework/util/AnnotatedTypes.java

```diff
@@ -1407,14 +1409,14 @@ private static boolean containsModifierImpl(AnnotatedTypeMirror type, Annotation
 
     private static Map<TypeElement, Boolean> isTypeAnnotationCache = new IdentityHashMap<>();
 
-    public static boolean isTypeAnnotation(AnnotationMirror anno) {
+    public static boolean isTypeAnnotation(AnnotationMirror anno, Class<?> cls, BaseTypeChecker checker) {
```

Comment:
I could see calling this method for annotations that may not be apart of a checker.  In those cases, the error shouldn't be issued.  In general, I would not expect an method in AnnotatedTypes to issue a warning.